fake_switches/brocade/command_processor/enabled.py

```diff
@@ -43,6 +43,14 @@ def do_show(self, *args):
             else:
                 self.write_line("Invalid input -> %s" % args[1])
                 self.write_line("Type ? for a list")
+        elif "route".startswith(args[1]):
+            if "static".startswith(args[2]):
+                routes = self.switch_configuration.static_routes
+                if routes:
+                    self.write_line("         Destination        Gateway         Port          Cost          Type Uptime src-vrf")
+                for n, r in enumerate(routes):
+                    self.write_line("%s        %s         %s" % (n+1, r.subnet, r.ip))
```

Comment:
New standard is using .format()

For table display, i suggest you use format's padding.  ie  '{0: <16}'.format('Hi')